Spring2016/questions/q04/q04.go

```diff
@@ -1,173 +1,85 @@
-package main
+package q04
 
-// Run the code online here: https://play.golang.org/
-
-import "fmt"
+// Run the code online here: https://play.golang.org/.
+// Or, run `go test` (see q04_test.go).
```

Comment:
When you've been working at google and need full sentence comments now :3

Reply:
Lol 💯% truuuu